lib/gen/translations/codegen_loader.g.dart

```diff
@@ -56,5 +57,21 @@ abstract class  LocaleKeys {
   static const days = 'days';
   static const months = 'months';
   static const fromDateToDate = 'fromDateToDate';
+  static const transactionFilterBy = 'transactionFilterBy';
+  static const transactionFrom = 'transactionFrom';
+  static const transactionTo = 'transactionTo';
+  static const transactionIncome = 'transactionIncome';
+  static const transactionExpenses = 'transactionExpenses';
+  static const transactionDate = 'transactionDate';
+  static const transactionTime = 'transactionTime';
+  static const transactionAmount = 'transactionAmount';
+  static const transactionParty = 'transactionParty';
+  static const transactionCategory = 'transactionCategory';
+  static const transactionAttachment = 'transactionAttachment';
+  static const transactionDescription = 'transactionDescription';
+  static const transactionTypeHere = 'transactionTypeHere';
+  static const transactionRecordIncome = 'transactionRecordIncome';
+  static const transactionRecordExpenses = 'transactionRecordExpenses';
+  static const groupsMyGroups = 'groupsMyGroups';
```

Comment:
Normally the translation keys where changed in my suggestion, it looks like you just used github commit!

This would break stuff you need to make sure that all the places where you used the keys are updated maybe with the IDE find and replace function